pkg/machinery/client/resolver/roundrobin.go

```diff
@@ -61,10 +62,19 @@ type roundRobinResolver struct {
 func (r *roundRobinResolver) start() error {
 	var addrs []resolver.Address //nolint:prealloc
 
-	for _, a := range strings.Split(r.target.Endpoint, ",") { //nolint:staticcheck
+	for _, addr := range strings.Split(r.target.Endpoint, ",") { //nolint:staticcheck
```

Comment:
how can we get to 'no port' scenario here?

Reply:
you're right, that wouldn't happen now.